.github/workflows/release_trial_frontend.yml

```diff
@@ -0,0 +1,79 @@
+
+name: Release Trial Frontend
+
+on:
+  push:
+    branches:
+      - trialfrontend01
+      - trialfrontend02
+      - trialfrontend03
+
+defaults:
+  run:
+    working-directory: prime-router
+
+jobs:
+  pre_job:
+    name: "Set Build Environment"
+    runs-on: ubuntu-latest
+    outputs:
+      env_name: ${{ steps.build_vars.outputs.env_name }}
+      version: ${{ github.ref_name }}
+    steps:
+      - name: Check out changes
+        uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f
```

Comment:
checkout was updated this week.  It should now be:
actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3